rules/anonymous-requests-to-kubelet-updated/raw.rego

```diff
@@ -0,0 +1,70 @@
+package armo_builtins
+
+deny[msga] {
+	kubelet_info := input[_]
+	kubelet_info.kind == "KubeletInfo"
+	kubelet_info.apiVersion == "hostdata.kubescape.cloud/v1beta0"
+	command := kubelet_info.data.cmdLine
+
+	contains(command, "--anonymous-auth")
```

Comment:
why not check for `"--anonymous-auth=true"`?

Reply:
For readability of the rule when taking every scenario it is easier to read it later

Reply:
also changed and removed unnecessary comments